index.js

```diff
@@ -142,7 +238,23 @@ function getAllMoviesReleasedAtOrBeforeYear() {}
  *  getBiggestBoxOfficeMovie(movies);
  *  //> "Incredibles 2"
  */
-function getBiggestBoxOfficeMovie() {}
+function getBiggestBoxOfficeMovie(movies) {
+//Return movies.title of the movie with the highest movies.boxOffice amount.
+//Convert movies.boxOffice into a number to compare to others.
+
+  let biggestMovie = 0;
```

Comment:
I would call this `biggestBoxOffice` as opposed to `biggestMovie` since it doesn't represent a `movie` object but rather a box office number.